src/priors/ioup.jl

```diff
@@ -24,45 +24,58 @@ where ``L`` is the `rate_parameter`.
 julia> solve(prob, EK1(prior=IOUP(2, -1)))
 ```
 """
-struct IOUP{elType,dimType,R} <: AbstractODEFilterPrior{elType}
-    wiener_process_dimension::dimType
-    num_derivatives::Int
+struct IOUP{elType,R} <: AbstractGaussMarkovProcess{elType}
+    d::Int
```

Comment:
Maybe rename these fields again. They should never be accessed with their name anyways.